Description: Allow guest session processes to signal and ptrace each other
Author: Tyler Hicks <tyhicks@canonical.com>

Index: lightdm-1.9.13/data/apparmor/abstractions/lightdm
===================================================================
--- lightdm-1.9.13.orig/data/apparmor/abstractions/lightdm	2013-10-30 17:34:00.000000000 -0500
+++ lightdm-1.9.13/data/apparmor/abstractions/lightdm	2014-04-02 13:47:09.651587353 -0500
@@ -71,6 +71,10 @@
 
   capability ipc_lock,
 
+  # allow processes in the guest session to signal and ptrace each other
+  signal peer=@{profile_name},
+  ptrace peer=@{profile_name},
+
   # silence warnings for stuff that we really don't want to grant
   deny capability dac_override,
   deny capability dac_read_search,
